gr-utils/modtool/cli/add.py

```diff
@@ -53,7 +53,6 @@ def cli(**kwargs):
         raise ModToolException('Missing or skipping relevant subdir.')
     get_blockname(self)
```

Comment:
I propose to move the name checking before requesting the block type. Otherwise you have to enter some parameters before the check is made and gr_modtool stopped

Reply:
That is a very good suggestion.  i'll have to take a look at where that should go.  For this PR, was just trying to get master consistent with 3.8 - perhaps that should be another PR.

Reply:
It is quite easy. Just move 
line 54 get_blockname(self)
between line 46 click.secho("GNU Radio module name identified: " + self.info['modname'], fg='green') 
and line 47 get_blocktype(self) 

That's it. I tested it this weekend.

Reply:
thanks!  yeah that wasn't so bad lol